### DISCUSSION TITLE
Which latest and stablest version of Log4cplus I should use on CentOS Linux 7 with GCC version 4.8.5 compiler

### DISCUSSION THREAD

**Discussion by GitHub121380** [log4cplus/log4cplus#621] [Q&A] [answered]
Hi,

My application need to be ported and recompiled on CentOS Linux 7 with GCC version 4.8.5 version which uses Log4cplus.
Pls let me know which is the latest version of log4cplus compatible for this system requirements?

I know about 2.x needs gcc version above 6.2.0, but the unsupported compilers is GCC prior to 4.8. So Could I use the log4cplus of 2.0.0 version on CentOS Linux 7 with GCC version 4.8.5?

Thanks for help!

**Comment by wilx** [comment, marked as answer]
It seems that GCC 4.8 does support C++11 (https://gcc.gnu.org/projects/cxx-status.html#cxx11). So, in theory, even log4cplus 2.1.1 could work. But I have not tested it.

The "Platform Support" list in the readme file is mainly about what has been tested at some point in time. It is not a list of minimal supported versions.

**Reply by GitHub121380** [reply]
ok, thank you!!!
